packages/core/src/pipeline/config/stages/evaluateVariables/ExecutionAndStagePicker.tsx

```diff
@@ -23,7 +24,10 @@ export function ExecutionAndStagePicker(props: IExecutionAndStagePickerProps) {
   const [showStageSelector, setShowStageSelector] = React.useState(false);
 
   const fetchExecutions = useData(
-    () => executionService.getExecutionsForConfigIds([pipeline.id], { limit: 100 }),
+    () =>
+      executionService.getExecutionsForConfigIds([pipeline.id], {
+        limit: SETTINGS.maxFetchHistoryOnEvaluateVariables ?? 100,
```

Comment:
Is there some way to keep 100 in one place?

Reply:
There's some code in packages/core/src/config/settings.ts at line 167 that sets default values for the configs if they are missing. I think that's the right place to put the 100 by default, rather than worrying about it here.

Reply:
Moved default value config to `packages/core/src/config/settings.ts` as per review comment. Thank you.